package.json

```
@@ -7,8 +7,8 @@
     "test": "standard && tap -J test/*.test.js",
     "test:report": "standard && tap -J --coverage-report=html test/*.test.js",
     "test:verbose": "standard && tap -J test/*.test.js -Rspec",
-    "load-data": "docker run --rm  -d --link fastify-postgres:postgres postgres:11-alpine psql -h postgres -d postgres -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);' -U postgres",
-    "postgres": "docker run --rm -d -p 5432:5432 --name fastify-postgres postgres:11-alpine"
+    "postgres": "docker run -p 5432:5432 --name fastify-postgres -e POSTGRES_PASSWORD=postgres -d postgres",
```

I would keep the verson
```suggestion
    "postgres": "docker run -p 5432:5432 --name fastify-postgres -e POSTGRES_PASSWORD=postgres -d postgres:11-alpine",
```